packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -589,6 +590,11 @@ Please file an issue at: https://github.com/shorebirdtech/shorebird/issues/new
   if (autoUpdate != null) {
     yamlContent.writeln('auto_update: $autoUpdate');
   }
+
+  final String? shorebirdPublicKeyEnvVar = Platform.environment['SHOREBIRD_PUBLIC_KEY'];
+  if (shorebirdPublicKeyEnvVar != null) {
+    yamlContent.writeln('patch_public_key: $shorebirdPublicKeyEnvVar');
```

Comment:
Do we need to quote the value?

Reply:
you don't need to quote string in yamls, unless we want to keep some style to them? No strong opinions here

Reply:
sg.  I think what got me thinking was wondering how much we should trust this env variable.  I suspect we should trust it fully.  But if it has a new line in it (by accident) that could cause invalid yaml, etc.) 🤷